content/300-guides/400-migrate-to-prisma/02-migrate-from-sequelize.mdx

```diff
@@ -0,0 +1,1217 @@
+```
+
+The respective connection URL would look as follows in Prisma:
+
+```sh file=.env
+DATABASE_URL="sqlserver://localhost:1433;database=blog-sequelize;user=alice;password=myPassword42;trustServerCertificate=true"
+```
+
+Note that if you are running on macOS, you must use `encrypt=DANGER_PLAINTEXT` to work around the current [TLS limitation](https://github.com/prisma/prisma/issues/4075):
+
+```
+DATABASE_URL="sqlserver://localhost:1433;database=blog-sequelize;user=alice;password=myPassword42;trustServerCertificate=true;encrypt=DANGER_PLAINTEXT"
+```
+
+</tab>
+
+<tab>
+
+Assume you have the following database connection details in `src/models/index.js`:
+
+```js file=src/models/index.js
+const sequelize = new Sequelize({
+  dialect: 'sqlite',
+  storage: '../../blog-sequelize.sqlite'
+});
+```
+
+The respective connection URL would look as follows in Prisma:
+
+```sh file=.env
+DATABASE_URL="file:./blog-sequelize.db"
+```
+
+</tab>
+
+</TabbedContent>
+
+### 2.3. Run Prisma's introspection
+
+With your connection URL in place, you can [introspect](../../concepts/components/introspection) your database to generate your Prisma models:
+
+```terminal copy
+npx prisma introspect
+```
+
+This creates the following Prisma models:
+
+```prisma file=prisma/schema.prisma
+model Categories {
+  id             Int              @id @default(autoincrement())
+  name           String
+  createdAt      DateTime
+  updatedAt      DateTime
+  PostCategories PostCategories[]
+}
+
+model PostCategories {
+  createdAt  DateTime
+  updatedAt  DateTime
+  CategoryId Int
+  PostId     Int
+  Categories Categories @relation(fields: [CategoryId], references: [id])
+  Posts      Posts      @relation(fields: [PostId], references: [id])
+
+  @@id([CategoryId, PostId])
+}
+
+model Posts {
+  id             Int              @id @default(autoincrement())
+  title          String
+  content        String?
+  published      Boolean?         @default(false)
+  createdAt      DateTime
+  updatedAt      DateTime
+  authorId       Int?
+  Users          Users?           @relation(fields: [authorId], references: [id])
+  PostCategories PostCategories[]
+}
+
+model Profiles {
+  id        Int      @id @default(autoincrement())
+  bio       String
+  createdAt DateTime
+  updatedAt DateTime
+  userId    Int?
+  Users     Users?   @relation(fields: [userId], references: [id])
+}
+
+model SequelizeMeta {
+  name String @id
+}
+
+model Users {
+  id        Int       @id @default(autoincrement())
+  name      String?
+  email     String    @unique
+  createdAt DateTime
+  updatedAt DateTime
+  Posts     Posts[]
+  Profiles  Profiles?
+}
+```
+
+The generated Prisma models represent your database tables and are the foundation for your programmatic Prisma Client API which allows you to send queries to your database. 
+We'll adjust the `createdAt` and `updatedAt` fields in our models. Sequelize doesn't add the `DEFAULT` constraint to `createdAt` when creating the tables in the database. 
```

Comment:
A few notes about this paragraph:

- I'd add a new subheading for this, smth like `### 2.4. Adjust `createdAt` and `updatedAt` fields
- Let's try to use "you" instead of "we" where possible
- Maybe link to a section in our docs where `@default(now())` and `@udpatedAt` are explained (`"You can read more about how Prisma handles `createdAt` and `updatedAt` functionality [here]()..."`)